src/PollingBlockTracker.ts

```diff
@@ -6,12 +6,13 @@ import { BaseBlockTracker, Provider } from './BaseBlockTracker';
 const createRandomId = getCreateRandomId();
 const sec = 1000;
 
-interface PollingBlockTrackerArgs {
-  provider: Provider;
-  pollingInterval: number;
-  retryTimeout: number;
-  keepEventLoopActive: boolean;
-  setSkipCacheFlag: boolean;
+export interface PollingBlockTrackerOptions {
+  provider?: Provider;
+  pollingInterval?: number;
+  retryTimeout?: number;
+  keepEventLoopActive?: boolean;
+  setSkipCacheFlag?: boolean;
+  blockResetDuration?: number | undefined;
```

Comment:
Nit: Another case of two ways to leave the option unset
```suggestion
  blockResetDuration?: number;
```

Reply:
Thanks!